src/components/Tool/FixSendNewsletter/FixSendNewsletter.tsx

```diff
@@ -143,110 +110,122 @@ const FixSendNewsletter: React.FC<Props> = ({
-                            firstName: '',
-                            lastName: '',
-                            primaryEmailAddress: {
-                              email: '',
-                            },
-                            optoutEnewsletter: false,
-                            deceased: false,
-                          }
-                        }
-                        key={contact.id}
-                        primaryAddress={
-                          contact.primaryAddress || {
-                            street: '',
-                            city: '',
-                            state: '',
-                            postalCode: '',
-                            source: '',
-                            createdAt: '',
-                          }
-                        }
-                        handleSingleConfirm={handleSingleConfirm}
-                        setContactFocus={setContactFocus}
-                      />
-                    ),
-                )}
-              </Grid>
-              <Grid item xs={12}>
-                <Box className={classes.footer}>
-                  <Typography>
+    <Box
+      className={classes.outer}
+      display="flex"
+      flexDirection="column"
+      data-testid="Home"
+    >
+      <Grid container className={classes.container}>
+        <Grid item xs={12}>
+          <Typography variant="h4">{t('Fix Send Newsletter')}</Typography>
+          <Divider className={classes.divider} />
+        </Grid>
+
+        <Grid item xs={12}>
+          <Box className={classes.descriptionBox}>
+            <Typography>
+              <strong>
+                {
+                  <Trans
+                    defaults="You have {{amount}} newsletter statuses to confirm."
+                    values={{
+                      amount: loading ? '...' : totalCount,
+                    }}
+                  />
+                }
+              </strong>
+            </Typography>
+            <Typography>
+              {t(
+                'Contacts that appear here have an empty Newsletter Status and Partner Status set to Financial, Special, or Pray. Choose a newsletter status for contacts below.',
+              )}
+            </Typography>
+          </Box>
+        </Grid>
+        {!loading && data && !!numberOfContacts ? (
+          <>
+            <ButtonHeaderBox mb={0}>
+              <Box>
+                <Typography>
+                  <Trans
+                    defaults="<i>Showing <bold>{{numberOfContacts}}</bold> of <bold>{{totalCount}}</bold></i>"
+                    shouldUnescape
+                    values={{
+                      numberOfContacts,
+                      totalCount,
+                    }}
+                    components={{ bold: <strong />, i: <i /> }}
+                  />
+                </Typography>
+              </Box>
+              {(loading || updating) && <LoadingSpinner firstLoad={true} />}
```

Comment:
Since on line 146, you check that it's not loading, you don't need to check if it's loadingon this line